app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
You can remove this entire app/views/devise/mailer directory. We're not sending any emails so all of these pages are not neccessary